l10n_it_fatturapa_in_notifications/tests/test_notifications_ftpa_in.py

```diff
@@ -116,11 +129,40 @@ def test_0_xml_import(self):
             cr, uid, notification.fatturapa_in_attachment_id.id)
         self.assertEqual(invoice_data.file_identifier, 'IT05979361218_002')
         self.assertEqual(invoice_data.sdi_identifier, 5199973)
-        #create accept notifications
+        # create accept notifications
         self.run_send_notif_wizard('accept', invoice_id)
         invoice = self.invoice_model.browse(cr, uid, invoice_id)
         attachment = self.notifications_model.browse(
             cr, uid, invoice.result_notification_id.id)
-        #check notification
+        # check notification
         xml_content = attachment.datas.decode('base64').decode('latin1')
         self.check_content(xml_content, 'IT05979361218_002_EC_002.xml')
+
+    def test_1_xml_import(self):
+        cr, uid = self.cr, self.uid
+        # import source invoice
+        res = self.run_import_invoice_wizard(
+            'test1', 'IT02780790107_11007.xml')
+        invoice_id = res.get('domain')[0][2][0]
+        # import metadata
+        # check metadata file name is case insensitive
+        res_notif = self.run_import_notif_wizard(
+            'IT02780790107_11007_MT_001.xml')
+        notif_id = res_notif.get('res_id')
+        notification = self.notifications_model.browse(cr, uid, notif_id)
+        self.assertEqual(notification.message_type, 'MT')
+        invoice_data = self.attach_model.browse(
+            cr, uid, notification.fatturapa_in_attachment_id.id)
+        self.assertEqual(invoice_data.file_identifier, 'IT02780790107_11007')
+        self.assertEqual(invoice_data.sdi_identifier, 5199979)
+        # create reject notifications
+        self.run_send_rej_notif_wizard('rejected', invoice_id)
+        invoice = self.invoice_model.browse(cr, uid, invoice_id)
+        attachment = self.notifications_model.browse(
+            cr, uid, invoice.result_notification_id.id)
+        # check notification
+        xml_content = attachment.datas.decode('base64').decode('latin1')
+        self.check_content(xml_content, 'IT02780790107_11007_EC_002.xml')
+        # check if rejected field is True and invoice is cancelled
+        self.assertEqual(invoice.rejected, True)
+        self.assertEqual(invoice.state, 'cancel')
```

Comment:
ah giustovero.. ma non fa nessun rise.. basta un assertTrue , siccome se non si puo' riportare in bozza non lo faccio e basta, non genero alcuna eccezione..